src/Command/CodeStudio/CodeStudioWizardCommand.php

```diff
@@ -189,8 +191,8 @@ protected function commandRequiresAuthentication(InputInterface $input): bool {
    */
   protected function validateEnvironment() {
     //$this->requireCloudIdeEnvironment();
-    if (!getenv('GITLAB_HOST')) {
-      throw new AcquiaCliException('The GITLAB_HOST environmental variable must be set.');
+    if (self::isAcquiaCloudIde() && !getenv('GITLAB_HOST')) {
+      throw new AcquiaCliException('The GITLAB_HOST environment variable must be set or the `--gitlab-host-name` option must be passed.');
```

Comment:
But you're not actually checking the option here? It seems like it could still fail even with --gitlab-host-name being passed.

Reply:
So we check `GITLAB_HOST` env variable only when its cloud ide. If not cloud ide (assuming local), then next execution is to fetch the hostname from options or from local config and there we throw exception.

Reply:
I don't see why we should limit it in that way. We need the host url. So, I'd check `!getenv('GITLAB_HOST') && !$this->input->getOption('gitlab-host-name')`